src/service.rs

```diff
@@ -359,6 +359,42 @@ impl ServiceFailureActions {
     }
 }
 
+/// Enum describing the service launch protection options.
+///
+/// See <https://learn.microsoft.com/ko-kr/windows/win32/api/winsvc/ns-winsvc-service_launch_protected_info>
+/// for more information.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[repr(u32)]
+pub enum ServiceLaunchProtected {
```

Comment:
It seems like they are inserting newline after each field so it's better to follow in this enum. Also, you could add doc comments for each field.

```suggestion
pub enum ServiceLaunchProtected {
    /// No launch protection. The service can be modified or replaced without restriction.
    None = Services::SERVICE_LAUNCH_PROTECTED_NONE,

    /// Launch protection for Windows components.
    Windows = Services::SERVICE_LAUNCH_PROTECTED_WINDOWS,

    /// A lighter version of Windows launch protection.
    WindowsLight = Services::SERVICE_LAUNCH_PROTECTED_WINDOWS_LIGHT,

    /// Launch protection used by antimalware (ELAM) services.
    AntimalwareLight = Services::SERVICE_LAUNCH_PROTECTED_ANTIMALWARE_LIGHT,
}
```